update-tags.mjs

```diff
@@ -0,0 +1,47 @@
+import { readFile, writeFile } from 'fs/promises'
+import { exec } from 'child_process'
+import { glob as baseGlob } from 'glob'
+import packageJSON from './package.json' assert { type: 'json' }
+
+const tagPath = `/v${packageJSON.version}/`
+
+async function glob(path) {
+  return await baseGlob(path, {
+    ignore: ['test/**', 'node_modules/**'],
```

Comment:
The `test` directory does not exist.

```suggestion
    ignore: ['node_modules/**'],
```

Alternatively, could we get the list of patterns to ignore from `.gitignore`?